unit_tests/tests/trigger/test_trigger_decoder.cpp

```diff
@@ -104,7 +103,10 @@ TEST(trigger, test1995FordInline6TriggerDecoder) {
 }
 
 TEST(misc, testGetCoilDutyCycleIssue977) {
-	EngineTestHelper eth(engine_type_e::FORD_ASPIRE_1996);
+	EngineTestHelper eth(engine_type_e::TEST_ENGINE);
```

Comment:
@FDSoftware do you want to PR happy part of this PR?

Reply:
https://github.com/rusefi/rusefi/pull/8189